### PR TITLE
Add docs section on CRL configuration

### PR DESCRIPTION
**Pull request by dopey**

### CODE REVIEW

**Review by tashian — CHANGES_REQUESTED** [review]
@dopey did you mean to add the PR template and triage workflow yml here?

File: src/pages/docs/step-ca/configuration.mdx [hunk 1]
```diff
@@ -238,6 +238,17 @@ CA's keys.
 
   - **valueDir**: directory to store the value log in (Badger specific).
 
+- **crl**: Certificate Revocation List(CRL) settings.
```
Comment (tashian) [inline src/pages/docs/step-ca/configuration.mdx]:
```suggestion
- **crl**: Certificate Revocation List(CRL) settings. (experimental)
```

File: src/pages/docs/step-ca/configuration.mdx [hunk 2]
```diff
@@ -238,6 +238,17 @@ CA's keys.
```
Comment (tashian) [inline src/pages/docs/step-ca/configuration.mdx:241]: Should we also update the example config to have a crl section with just `"enable": false` in it?

**Review by tashian — CHANGES_REQUESTED** [review]

File: src/pages/docs/step-ca/configuration.mdx (hunk 2 above)
Comment (dopey) [reply src/pages/docs/step-ca/configuration.mdx:241]: sure. I've not seen an actual config using this stuff yet, so was hesitant to add anything to the example.

**Review by tashian — APPROVED** [review]

**Pull request merged**